homeassistant/components/openhome/media_player.py

```diff
@@ -19,34 +20,58 @@
     SUPPORT_VOLUME_SET,
     SUPPORT_VOLUME_STEP,
 )
-from homeassistant.const import STATE_IDLE, STATE_OFF, STATE_PAUSED, STATE_PLAYING
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    STATE_IDLE,
+    STATE_OFF,
+    STATE_PAUSED,
+    STATE_PLAYING,
+)
+from homeassistant.helpers import config_validation as cv, entity_platform
+
+from .const import ATTR_PIN_INDEX, DATA_OPENHOME, SERVICE_INVOKE_PIN
+
+OPENHOME_PIN_SCHEMA = vol.Schema(
+    {
+        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
+        vol.Required(ATTR_PIN_INDEX): cv.positive_int,
+    }
+)
 
```

Comment:
We don't need this. If we pass a dict when registering the entity service the service helper will create an entity schema for us.
```suggestion
```